Gopkg.lock

```diff
@@ -99,6 +99,17 @@
   revision = "0ca9ea5df5451ffdf184b4428c902747c2c11cd7"
   version = "v1.0.0"
```

Comment:
Please as a general good practice let's include reason for re vendoring in the commit message

Reply:
`Gopkg.lock` is auto-generated by godep